### ISSUE TITLE
could not modify png

### ISSUE THREAD

**Issue by Mulicia** [marchyman/GeoTag#54]
in previous version geotag can modify png file, but after update to 5,it no longer change gps of png files

**Comment by marchyman** [comment]
That went away in version 4.8.  The commit message for 17 Mar 2020 says:

`macos/finder can't read the metadata from PNG files. Gray them out instead
of frustrating users wondering why their updates didn't take.`

I'll have to check to see of the macOS/Finder limitation still exists.

**Comment by marchyman** [comment]
Interesting.  I took a screenshot on my phone to create a PNG file.  That went to my Photos Library.  I imported the screenshot into GeoTag from the Photos Library and assigned a location and "saved" the image.  I then exported the image from Photos as a PNG allowing location info.  The result: Finder and Preview had no problems showing the image.

I removed the PNG limitation in a test version of GeoTag.  Dragging the image into GeoTag shows the location.  I can change and save the location, too.  It looks like whatever restriction existed in 2020 no longer exists.  PNG support will be back in the next release.  The change has been committed to the "fixes" branch.

Thank you for your report.
